frontend/test_setup/fixtures/filters.json

```diff
@@ -69,18 +69,6 @@
                 "displayed": true,
                 "readonly": false,
                 "type": "primary"
-              },
-              {
```

Comment:
why removing this from the test case?

Reply:
Shoot, this was pulled by mistake. Reverting